java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -451,26 +451,26 @@
 
     //Lists of items that describe the Layout, and allow it to be drawn
     //Each of the items must be saved to disk over sessions
-    public ArrayList<PositionableLabel> backgroundImage = new ArrayList<>();    //background images
-    public ArrayList<SensorIcon> sensorImage = new ArrayList<>();               //sensor images
-    public ArrayList<SignalHeadIcon> signalHeadImage = new ArrayList<>();       //signal head images
-    public ArrayList<LocoIcon> markerImage = new ArrayList<>();                 //marker images
-    public ArrayList<PositionableLabel> labelImage = new ArrayList<>();         //positionable label images
-    public ArrayList<AnalogClock2Display> clocks = new ArrayList<>();           //fast clocks
-    public ArrayList<MultiSensorIcon> multiSensors = new ArrayList<>();         //multi-sensor images
-
-    public ArrayList<LayoutTurnout> turnoutList = new ArrayList<>();        //LayoutTurnout list
-    public ArrayList<TrackSegment> trackList = new ArrayList<>();           //TrackSegment list
-    public ArrayList<PositionablePoint> pointList = new ArrayList<>();      //PositionablePoint list
-    public ArrayList<LevelXing> xingList = new ArrayList<>();               //LevelXing list
-    public ArrayList<LayoutSlip> slipList = new ArrayList<>();              //LayoutSlip list
-    public ArrayList<LayoutTurntable> turntableList = new ArrayList<>();    //LayoutTurntable list
-
-    public ArrayList<SignalHeadIcon> signalList = new ArrayList<>();                //Signal Head Icons
-    public ArrayList<MemoryIcon> memoryLabelList = new ArrayList<>();               //Memory Label List
-    public ArrayList<BlockContentsIcon> blockContentsLabelList = new ArrayList<>(); //BlockContentsIcon Label List
-    public ArrayList<SensorIcon> sensorList = new ArrayList<>();                    //Sensor Icons
-    public ArrayList<SignalMastIcon> signalMastList = new ArrayList<>();            //Signal Mast Icons
+    public ArrayList<PositionableLabel> backgroundImage = new ArrayList();    //background images
```

Comment:
We want to restore the diamonds in the List constructors throughout this class unless their presence causes a compiler issue. If it does cause a compiler issue, please comment on it. See https://stackoverflow.com/questions/4166966/what-is-the-point-of-the-diamond-operator-in-java-7#4167148 for more information.